src/js/components/awardv2/contract/ContractContent.jsx

```diff
@@ -33,6 +35,13 @@ export default class ContractContent extends React.Component {
                         {this.props.selectedAward.id}
                     </div>
                 </div>
+                <div className="award__row agencyRecipient">
```

Comment:
You probably want to use the standard CSS naming convention here. So agencyRecipient should become agency-recipient.